styles/style.css

```diff
@@ -0,0 +1,111 @@
+/*.grid-container {
+  display: inline-grid;
+  grid-template: 1fr 1fr 1fr 1fr 1fr/ 1fr ;
+  grid-gap: 15px;
+}*/
+
+nav ul li{
+  display:inline-block;
+  width: 400px;
+  height: 250px
+  padding: 5px;
+  text-align: center;
+  list-style: none;
+}
+section {
+  text-align: center;
+  padding: 15px;
+}
+section h1{
+  font-size: 3em;
+  text-align: center;
+  font-family: sans-serif;
+  padding: 15px;
+  background-color: #DCCFEC
+}
+section sub {
+  display:inline-block;
+  width: 1320px;
+  height: 250px
+  padding: 10px;
+  font-size: 1.5em;
+  font-family: sans-serif;
+  text-align: center;
+  padding-bottom:2.5em;
+
+}
+
+a:link  {
+  display: inline-block;
+  height: 20px;
+  width: 200px;
+  text-decoration: none;
+  border-radius: 10px;
+  font-family: sans-serif;
+  font-size: 20px;
+  font-weight: lighter;
+  text-align: center;
+  border: 1px dotted;
+  text-align: center;
+  padding-top: 1em;
+  padding-bottom: 1em;
+
+}
+hr {
+    display: block;
+    height: 1px;
+    border: 0;
+    border-top: 1px solid #ccc;
+    margin: 1em 0;
+    padding-top: 2em;
+    padding-bottom: 2em;
+}
+.container {
+    width: 1200px;
+    align: center;
+}
+
+.quote {
+    border-top-left-radius: 8px;
+    border-top-right-radius: 8px;
+    color: #938D8D;
+    margin: 10;
+    padding: 20px;
+    font-size: 25px;
+    font-style: italic;
+}
+
+.name {
+    border-bottom-left-radius: 8px;
+    border-bottom-right-radius: 8px;
+    color: #000;
+    font-size: 20px;
+    padding: 20px;
+    text-align: right;
+    content: " ";
+}
+blockquote::before{
+  content: open-quote;
+}
+blockquote::after{
+  content: close-quote;
+}
+section h2 ul{
+  display:inline-block;
+  text-align: center;
+  list-style: none;
+}
+.container{
+  display: flex;
+  list-style: none;
+}
+
+}
+hr {
```

Comment:
Similar wrapping/responsive behavior as above. Try flex or grid with a wrap property. For this area, I ended up nesting display elements in each other to achieve something like the wireframe look.